bioconda.py

```diff
@@ -1,6 +1,9 @@
 """Bioconda-related functionality"""
 
-from utils import clean_github_link, find_all_paths
+import os
+
+from bioconda_utils.recipe import Recipe
+from utils import find_all_paths, nested_dictionary_extract
 
 
 def generate_bioconda_source_links(filepath):
```

Comment:
Check the `filepath` documentation. I think it might need to be updated.

Reply:
Done.